### PR TITLE
add chromatic to config

### PR DESCRIPTION
**Pull request by cellog**
# Description

This PR shows how we would enable chromatic. It does not set it up in CI yet, as we need to handle #1856 first.

To use, run with:

```
CHROMATIC_APP_CODE=<our app code> npm run chromatic
```

Replace `<our app code>` with the app code from https://chromaticqa.com (log in with github)

It only enables chromaticqa in unlock-app for now

# Issues

Refs #870 
Refs #1856 

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/.storybook/config.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 import { configure, addDecorator } from '@storybook/react'
+import 'storybook-chromatic'
 import StoryRouter from 'storybook-react-router'
+
```
Comment (julien51) [inline unlock-app/.storybook/config.js]: nit: whitespace?

Comment (cellog) [reply unlock-app/.storybook/config.js]: I have gotten in the habit of putting a space between node_modules and internal imports. I'll remove this

**Comment by cellog** [comment]
> Split the PR to add the dep ;)

I will, this is a draft, just want to make sure it works and is what we are looking for first. Would appreciate another look, when you have the time

File: .circleci/config.yml [hunk 2]
```diff
@@ -40,6 +40,32 @@ jobs:
...
+    steps:
+      - checkout
+      - run:
+          name: Pull Unlock Image
+          command: scripts/pull-image.sh unlock "$CIRCLE_SHA1"
+      - run:
+          name: Creator Dashboard stories
+          command: scripts/storybook.sh unlock-app
+
+  unlock-app-chromatic-master:
```
Comment (julien51) [inline .circleci/config.yml]: I think rather than have 2 jobs you should push the logic of checking if we are on master down to `scripts/storybook.sh`

Comment (julien51) [reply .circleci/config.yml]: You can pass the branch name using `$CIRCLE_BRANCH` and inside your script, if the branch is master, then you would append `--auto-accept-changes`.

This way we are as decouple as possible from the CI tool we use.

Comment (cellog) [reply .circleci/config.yml]: thanks! this is the piece I needed.

**Comment by cellog** [comment]
@julien51 PTAL

**Pull request merged**